storybook/qmlTests/tests/tst_SwapInputPanel.qml

```diff
@@ -274,29 +274,6 @@ Item {
             verify(!maxTagButton.text.endsWith("ETH"))
         }
 
-        function test_clickingMaxButton() {
```

Comment:
Just skip the test with the same TODO you did for the code

Reply:
No, I really meant `skip("MESSAGE...")`, not comment out :)

See https://doc.qt.io/qt-5/qml-qttest-testcase.html#skip-method